params/protocol_params.go

```diff
@@ -143,11 +143,12 @@ const (
 	Bls12381MapG1Gas          uint64 = 5500   // Gas price for BLS12-381 mapping field element to G1 operation
 	Bls12381MapG2Gas          uint64 = 110000 // Gas price for BLS12-381 mapping field element to G2 operation
 
-	Bw6761G1AddGas   uint64 = 600   // Price for BW6-761 elliptic curve G1 point addition
-	Bw6761G1MulGas   uint64 = 12000 // Price for BW6-761 elliptic curve G1 point scalar multiplication
-	Bw6761G2AddGas   uint64 = 4500  // Price for BW6-761 elliptic curve G2 point addition
-	Bw6761G2MulGas   uint64 = 55000 // Price for BW6-761 elliptic curve G2 point scalar multiplication
-	Bw6761PairingGas uint64 = 23000 // Price for BW6-761 elliptic curve pairing check
+	Bw6761G1AddGas            uint64 = 600   // Price for BW6-761 elliptic curve G1 point addition
+	Bw6761G1MulGas            uint64 = 12000 // Price for BW6-761 elliptic curve G1 point scalar multiplication
+	Bw6761G2AddGas            uint64 = 4500  // Price for BW6-761 elliptic curve G2 point addition
+	Bw6761G2MulGas            uint64 = 55000 // Price for BW6-761 elliptic curve G2 point scalar multiplication
+	Bw6761PlonkProofVerifyGas uint64 = 21000 // Price for BW6-761 elliptic curve PlonK proof verification
```

Comment:
Given that gas does matter _somewhat_ just in terms of feasibility of computation, I would suggest working out roughly how many of the other precompiles _would_ be used in this and then summing their gas costs. Give or take a bit. I would expect it to certainly be the most expensive of all of these, and by quite a bit.

Reply:
That should get revealed when I finally run the verification on my local chain. I'll then use the discovered value here.

Reply:
Since there's an ACK for this PR, I'll merge it and open an issue for this gas value

Reply:
I just realized I can't open an issue in this repo, there's no such option. I can only see the issues tab when I go into the upstream repo.